tracer/src/Datadog.Tracer.Native/clr_helpers.cpp

```diff
@@ -1004,4 +1004,41 @@ HRESULT FunctionLocalSignature::TryParse(PCCOR_SIGNATURE pbBase, unsigned len, s
     return S_OK;
 }
 
+HRESULT HasAsyncStateMachineAttribute(const ComPtr<IMetaDataImport2>& metadataImport, const mdMethodDef methodDefToken)
```

Comment:
Why can't this method just return `bool`? We aren't exposing it as part of a COM interface, are we?

Reply:
What should we do if `GetCustomAttributeByName` returns `E_FAIL`?

Reply:
Fixed in 5a103fbe57cedbc9e13568d4372c82a921c46e26 by using out bool arg